android/build.gradle

```diff
@@ -22,10 +22,10 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 
 android {
-    compileSdkVersion project.hasProperty('compileSdkVersion') ? rootProject.ext.compileSdkVersion : 30
+    compileSdkVersion project.hasProperty('compileSdkVersion') ? rootProject.ext.compileSdkVersion : 32
```

Comment:
Capacitor 3's target & compile sdkVersion is 30. So if you need it, you will have to wait for a Capacitor4-compatible version.

Reply:
The new payment methods was added in v19.1.0 of Android Stripe SDK (https://github.com/stripe/stripe-android/pull/4489).
Unfortunately compileSdkVersion was bumped in 18.2.0 (https://github.com/stripe/stripe-android/commit/705390657782b2ce820304e92a536105c774db12).

Maybe there is a way to choose a SKD version based on compileSdkVersion of parent project?